CHANGELOG.md

```diff
@@ -4,6 +4,23 @@ All notable changes to this project will be documented in this file.
 Each new release typically also includes the latest modulesync defaults.
 These should not affect the functionality of the module.
 
+## [v4.1.0](https://github.com/voxpupuli/puppet-gitlab_ci_runner/tree/v4.1.0) (2021-11-03)
+
+[Full Changelog](https://github.com/voxpupuli/puppet-gitlab_ci_runner/compare/v4.0.0...v4.1.0)
+
+**Implemented enhancements:**
+
+- Make session\_server section configurable [\#132](https://github.com/voxpupuli/puppet-gitlab_ci_runner/pull/132) ([baurmatt](https://github.com/baurmatt))
+
+**Closed issues:**
+
+- Make session\_server section configurable [\#131](https://github.com/voxpupuli/puppet-gitlab_ci_runner/issues/131)
```

Comment:
I'd be tempted to mark this `skip-changelog` and regenerate.  No need to repeat ourselves too much in the changelog.

Reply:
Good idea! I've added the label and regenerated the changelog.